packages/react-components/react-nav-preview/src/components/NavItem/useNavItem.ts

```diff
@@ -53,6 +53,11 @@ export const useNavItem_unstable = (props: NavItemProps, ref: React.Ref<HTMLAnch
       }),
       { elementType: 'a' },
     ),
+    icon: slot.optional(icon, {
+      renderByDefault: false,
```

Comment:
undefined is the default which translates to false, this can be removed.
```suggestion
```

Reply:
updated :-)